src/pages/my-resources/MyResources.constants.tsx

```diff
@@ -37,5 +39,10 @@ export const tabsData: TabsData = {
     title: 'myResourcesPage.tabs.attachments',
     content: <AttachmentsContainer />,
     icon: <AttachFileIcon />
+  },
+  categories: {
+    title: 'myResourcesPage.tabs.categories',
+    content: <CategoriesContainer />,
+    icon: <CategoryIcon />
   }
 }
```

Comment:
![image](https://github.com/ita-social-projects/SpaceToStudy-Client/assets/102412173/94524f4d-7028-4707-a381-537b8e1310fe)

we don't need this line in figma design?

Reply:
done